docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -80,11 +76,12 @@ az rg ls
 az vm ls
 ```
 
+Don't include az as part of the command.
```

Comment:
Place `az` in preformat quotes ` `` ` so that it's clear that it's something users would type as part of a command. CLI commands are always written in backticks unless they are a link out to reference documentation.